src/generator/paths.ts

```diff
@@ -81,7 +81,7 @@ export const getOpenApiPathsObject = (
           operationId: mutationPath,
           summary,
           description,
-          tags: tag ? [tag] : undefined,
+          tags: tags ? tags : tag ? [tag] : undefined,
```

Comment:
Nit: nested ternaries are confusing 😅

```suggestion
          tags: tags ?? tag ? [tag] : undefined,
```

Reply:
Will push a fresh commit